core/api/Cargo.toml

```diff
@@ -10,6 +10,7 @@ async-graphql = { version = "3.0", features = ["tracing"] }
 cita_trie = "3.0"
 jsonrpsee = { version = "0.6", features = ["http-server", "macros"] }
 parking_lot = "0.11"
+ethereum = "0.10.0"
```

Comment:
Remove this, all the types used from crate ethereum are re-exported from protocol.